plugins/modules/ldap_attrs.py

```diff
@@ -309,16 +326,18 @@ def main():
 
     # Instantiate the LdapAttr object
     ldap = LdapAttrs(module)
+    old = None
+    new = None
```

Comment:
The nit picker in me would like to suggest a different name as `new` is a Python keyword (in the context of classes), just for the sake of differentiating from that.

Reply:
I don't think `new` is a keyword in Python. The language reference does not list it as a keyword: https://docs.python.org/3/reference/lexical_analysis.html#keywords

Reply:
In python it's not a keyword. When switching back and forth between languages that might still be confusing though, therefore I changed all occurences to `[old|new]_attrs`

Reply:
Again my jumping the gun. I was thinking of the `__new__` method in classes. Anyways, I am happy that the vars now have more meaning in their names.